### DISCUSSION TITLE
Yoast SEO Plugin

### DISCUSSION THREAD

**Discussion by jan10** [craftcms/cms#9553] [Plugin ideas]
Add the official Yoast SEO plugin/extension to the Craft Plugin Store: https://yoast.com/yoast-seo-platforms/your-platform/

Yoast SEO is the number 1 plugin on wordpress

**Comment by Wiejeben** [comment]
It is up to Yoast to publish a plugin for Craft, this would probably require them to rewrite their plugin available from scratch. See https://plugins.craftcms.com/categories/seo for all the SEO and accessibility related plugins. Personally I would recommend using SEOmatic if you are looking for something comparable to Yoast.
